form-runner/jvm/src/main/resources/apps/fr/components/view.xsl

```diff
@@ -488,7 +488,7 @@
             xxf:element="nav"
             model="fr-form-model"
             ref=".[not(xxf:property(string-join(('oxf.fr.detail.hide-header', fr:app-name(), fr:form-name()), '.')))]"
-            class="navbar navbar-expand-lg navbar-dark bg-dark position-sticky">
+            class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
```

Comment:
This causes the content of the page to be hidden by the navbar. The `<body>` needs padding, for example, in landing.less:

```css
body.orbeon {
  padding-top: 52px;
}
```

Reply:
I added it into the landing.less, but nothing happened. So, I just put it in landing.xhtml for now.